File: 344-reverse-string/344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int n = s.size();
        char ch;
        for(int i=0;i<n/2;i++)
        {
            ch = s[i];
            s[i] = s[n-1-i];
            s[n-1-i] = ch;
            
        }
        
    }
};

File: 344-reverse-string/344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        
        int n = s.length;
        int i =0;
        int  j = n-1;
        char c;
        while(i<j)
        {
           c = s[i];
            s[i] = s[j];
            s[j] = c;
            i++;
            j--;
        }
        
    }
}